Source/WebKit/UIProcess/API/mac/WKWebViewMac.mm

```diff
@@ -1583,6 +1583,23 @@ - (NSEdgeInsets)_obscuredContentInsets
     );
 }
 
+- (NSColor *)_overrideTopScrollEdgeEffectColor
+{
+    return _overrideTopScrollEdgeEffectColor.get();
+}
+
+- (void)_setOverrideTopScrollEdgeEffectColor:(NSColor *)color
+{
+    if (_overrideTopScrollEdgeEffectColor == color || [_overrideTopScrollEdgeEffectColor isEqual:color])
```

Comment:
`_overrideTopScrollEdgeEffectColor == color` iff `[_overrideTopScrollEdgeEffectColor isEqual:color]`, right?

Reply:
Oh, this pattern (`a == b || [a isEqual:b]`, for ObjC objects `a` and `b`) covers both the cases where `a` and `b` are `nil`, and when they're both non-null but equal by value.